detailed_docs/Wiki/0249_capabilities.md

```diff
@@ -0,0 +1,135 @@
+## Feature 1: As a mobile application, I want to get HMI capabilities from SDL after Ignition OFF/ON cycle so that I can interact with the HMI efficiently
+
+**Background**
+
+**Given** App requests HMI Capabilities with one of the following RPC
+  * UI.GetSupportedLanguages
+  * UI.GetCapabilities
+  * VR.GetSupportedLanguages
+  * VR.GetCapabilities
+  * TTS.GetSupportedLanguages
+  * TTS.GetCapabilities
+  * Buttons.GetCapabilities
+  * VehicleInfo.GetVehicleType
+  * RC.GetCapabilities
+  * UI.GetLanguage
+  * VR.GetLanguage
+  * TTS.GetLanguage
+#### Scenario 1: SDL Provides HMI capabilities persisted in `HMICapabilitiesCacheFile`
+
+**Given** App registers after Ignition OFF/ON cycle
+
+**When** App requests HMI Capabilities
+
+**Then** SDL checks `HMICapabilitiesCacheFile` in `AppStorageFolder` in [smartDeviceLink.ini](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/smartDeviceLink.ini)  
```

Comment:
my idea was 
**GIVEN**(situation/condition) - is like we normally have **_pre-condition_**
**WHEN**(user action) - most frequently in our case it's an **_application request_**
**THEN**(result) statements - this is the most tricky, because 

- from the standpoint of a user/client - here should be `something that comes out of the system (report, user interface, message)`

- from the standpoint of SDL contributors - here should be recorded **_SDL behaviour_**

In this attempt of mine, I was searching for some balance here, trying to please both sides.